js/main.js

```diff
@@ -1,2 +1,114 @@
-$(function () {
+$(function() {
+  let humanScores = new Array
```

Comment:
@rlou88 for performance reasons, it's better to initialize an array literally as opposed to using the constructor. Of course, there are some uses cases for the constructor, like setting a size upon declaration. 

By default, a literal declaration `let arr = []` will create an instance of a dynamic array. If you are interested in a fixed-size array, check here for [typed arrays](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays)